src/main/java/de/kaleidox/e2uClaim/config/PlayerPreferenceManager.java

```diff
@@ -0,0 +1,69 @@
+package de.kaleidox.e2uClaim.config;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Objects;
+import java.util.UUID;
+import java.util.concurrent.ConcurrentHashMap;
+
+import de.kaleidox.e2uClaim.E2UClaim;
+import de.kaleidox.e2uClaim.interfaces.Initializable;
+import de.kaleidox.e2uClaim.interfaces.Terminatable;
+
+import org.bukkit.configuration.file.FileConfiguration;
+
+public enum PlayerPreferenceManager implements Initializable, Terminatable {
+    INSTANCE;
+
+    private final Map<UUID, Map<String, String>> playerPropertyValueMap = new ConcurrentHashMap<>();
+    private final Map<String, String> defaultValues = new ConcurrentHashMap<>();
+
+    @Override
+    public void init() throws Throwable {
+        FileConfiguration config = E2UClaim.getConfig("userPreferences");
+        FileConfiguration defaults = E2UClaim.getConfig("config");
+        defaultValues.put("autoLock", Boolean.toString(defaults.getBoolean("defaults.auto-lock")));
```

Comment:
You still hardcode value paths. You should iterate over all values of section `defaults` for `config.yml`.